ps.go

```diff
@@ -0,0 +1,92 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+	"github.com/opencontainers/runc/libcontainer"
+)
+
+var psCommand = cli.Command{
+	Name:  "ps",
+	Usage: "lists containers started by runc",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
```

Comment:
We can use `--root` to change the state directory for different containers, I think we may need some constant directory for information that `runc ps` needs.